Model/Service/Facet/BuildWebFacetService.php

```diff
@@ -113,18 +121,21 @@ public function getFacets(): Facet
 
         try {
             $this->populateFilters($includeFilters);
+        } catch (FacetValueException $e) {
+            $this->debugWithSource('Cannot map filters, error message: %s', [$e->getMessage()]);
```

Comment:
I don't think we need to debug log it and exception log it. Simply log the exception and collapse both Exception blocks.